docs/leidraad/richtlijn-sjabloon-v1.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Sjabloon voor richtlijnen"
+tags: []
+---
+
+# Sjabloon voor richtlijnen, versie 1
+
+Vervang het kopje hierboven door "Richtlijn: _titel van de richtlijn_", bijvoorbeeld "Richtlijn: geautomatiseerd testen".
+
+Begin met een beschrijving van de richtlijn zonder deze te motiveren (dat komt in de volgende sectie).
+
+Leg termen uit en link deze (alleen bij de eerste keer dat ze gebruikt worden) naar openbare glossaries zoals de [Glossary of web terms](https://developer.mozilla.org/en-US/docs/Glossary) van het Mozilla Developer Network (MDN) of het [Online Browsing Platform](https://www.iso.org/obp/ui/en/#home) van de International Standards Organisation (ISO).
+
+Gebruik informele taal. Je kan de lezer aanspreken met "je". Afkortingen schrijf je de eerste keer uit met de afkorting tussen haakjes erachter.
+
+## Rationale: Waarom deze richtlijn?
+
+Vervang in het kopje hierboven "deze richtlijn" door de titel van de richtlijn.
+
+Motiveer waarom de richtlijn bijdraagt aan het realiseren van het principe waar deze onder valt. Noem ook risico's die optreden als de richtlijn niet wordt toegepast.
+
+## Doelgroep: Wie zijn er betrokken bij deze richtlijn?
```

Comment:
Moeten we ergens een centrale lijst van doelgroepen maken? Anders lopen we misschien het risico dat we bij elke richtlijn net iets andere benamingen gebruiken.